graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH8.kt

```diff
@@ -0,0 +1,90 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctionalElements
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+
+@BEH(GraphBehaviours.CreateDirectedGraph, GraphFunctionalElements.DirectedGraph, """
+    Create a directed graph
+""")
+class BEH8 : GraphScenario() {
+
+    @Eac("A directed graph is a graph that only contains directional edges",
+        """When a directional edge is declared a path can only be walked in one order
+        """)
+    fun directionalEdge() {
+        given {
+            thereIsADirectedGraph(graphA) {
+                edge('a', 'b')
+            }
+        }
+        then {
+            pathExists(graphA, from = 'a', to = 'b')
+            noPathExists(graphA, from = 'b', to = 'a')
+        }
+    }
+
+    @Eac("A graph may any positive number of vertices")
```

Comment:
⛏️ I'm not entirely sure whether this is an EAC, but anyway "contain" is missing.